src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -60,6 +61,7 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web create', 'name', options_list=('--name', '-n'), help='name of the new webapp')
 register_cli_argument('appservice web create', 'plan', options_list=('--plan', '-p'), completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                       help="name or resource id of the app service plan. Use 'appservice plan create' to get one")
+register_cli_argument('appservice web browse', 'start_trace', action='store_true', help='start live tracing')
```

Comment:
As a point of reference/prior art, the Serverless CLI has a command called `invoke` (which is effectively the same as `browse` here), which has a flag called `--logs` (aliased as `-l`), which allows immediately opening the log stream after running a cloud function.

I love to propose that we call this arg the exact same. "logs" is very recognizable, and is easy to associate with the "long form" command (i.e. `appservice logs tail`).